### ISSUE TITLE
User sent to the second-factor screen upon an unauthorised attempt of accessing a resource

### ISSUE THREAD

**Issue by omar-nahhas** [authelia/authelia#816]
Hi, I am trying using authelia to integrate with Gsuite LDAP. Thank you for your hard work. It is the simplest way I have found to set up an LDAP authentication proxy. I have found though, that if you want to use only the first-factor authentication feature (without password reset or second factor) is not possible to disable the formers cleanly.

**The problem:**
When a user authenticates with its username/password **but** is not granted access to the specific resource because it does not belong the group specified on the subject policy. The user is redirected to the second-factor screen.

![auth-1](https://user-images.githubusercontent.com/551844/78240032-58cee980-74d6-11ea-9f48-01151516884a.png)

This is the access control section:

```
access_control:
  default_policy: deny
  rules:
    - domain: auth.example.org
      policy: bypass
    - domain: web.example.org
      policy: one_factor
      subject: "group:group2"
```
The user exists on LDAP but it does not belong to group2. When the user belongs to group2, the user is granted access to the requested resource.

**What would be expected:**
The users should be redirected to a (configurable) 403 error page, or back to the log-in screen.

**Comment by clems4ever** [comment]
Hello @omar-nahhas , Good to know that Authelia fits your needs. Fortunatly, reset password disabling will be worked on in a couple of days. You should check for updates in https://github.com/authelia/authelia/issues/387.

Regarding your issue, can you confirm you use the latest version of Authelia?

**Comment by james-d-elliott** [comment]
@omar-nahhas I've solved this in a dev branch of Authelia. It will be merged into the 4.14.2 release.

**Comment by omsanchez-iov42** [comment]
Awesome
